houston/queries.go

```diff
@@ -306,7 +306,7 @@ var (
 		  id
 		  label
 		  description
-		  users {
+		  systemUsers {
```

Comment:
@andriisoldatenko I think we'll want the `workspaceUsers` query here. `systemUsers` is intended to be called via system admins and requires those permissions. `workspaceUsers` should be the equivalent of the old one.

Reply:
@schnie fixed, do you know where i can test it?
cc / @petedejoy